<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RiverMetricsDockWidgetBase</class>
 <widget class="QDockWidget" name="RiverMetricsDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RiverMetrics</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QComboBox" name="vectorCombo"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="validator">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="graph">
      <property name="text">
       <string>Graph</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <widget class="QDoubleSpinBox" name="stepSpin">
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="minimum">
       <double>10.000000000000000</double>
      </property>
      <property name="maximum">
       <double>99999.000000000000000</double>
      </property>
      <property name="value">
       <double>1000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Step value</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="validate">
      <property name="text">
       <string>Validate geom</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/plugins/RiverMetrics/valid.png</normaloff>:/plugins/RiverMetrics/valid.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Shift value</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="shiftSpin">
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="minimum">
       <double>10.000000000000000</double>
      </property>
      <property name="maximum">
       <double>99999.000000000000000</double>
      </property>
      <property name="value">
       <double>100.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QFrame" name="frame_for_plot">
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
